### ISSUE TITLE
rocAL - rocalGetImageName & rocalGetImageLabels function issue

### ISSUE THREAD

**Issue by hansely** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#952]
Tested with [rocAL_basic_test](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/utilities/rocAL/rocAL_basic_test/rocAL_basic_test.cpp#L190)

The rocalGetImageName() function doesn't return a single image file(It returns the whole batch)
The rocalGetImageLabel() function returns 0 for all the images.

```
$ ./rocAL_basic_test
Usage: image_augmentation <image_dataset_folder> <label_text_file_path> <test_case:0/1> <processing_device=1/cpu=0>  decode_width decode_height <gray_scale:0/rgb:1> decode_shard_counts

$ ./rocAL_basic_test ../../../../data/images/AMD-tinyDataSet/ "" 0 0 22
>>> Running on CPU
OK: loaded 80 kernels from libvx_rpp.so
Augmented copies count 1
output width 224 output height 896 color planes 3
#### Started test id 0
Available images = 256
>>>>> Going to process 256 images , press a key
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 0
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 0
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 0
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 0

name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 0
name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 0
name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 0
name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 0

name AMD-tinyDataSet_0126.JPEGAMD-tinyDataSet_0002.JPEGAMD-tinyDataSet_0047.JPEGAMD-tinyDataSet_0135.JPEG label 0
name AMD-tinyDataSet_0126.JPEGAMD-tinyDataSet_0002.JPEGAMD-tinyDataSet_0047.JPEGAMD-tinyDataSet_0135.JPEG label 0
name AMD-tinyDataSet_0126.JPEGAMD-tinyDataSet_0002.JPEGAMD-tinyDataSet_0047.JPEGAMD-tinyDataSet_0135.JPEG label 0
name AMD-tinyDataSet_0126.JPEGAMD-tinyDataSet_0002.JPEGAMD-tinyDataSet_0047.JPEGAMD-tinyDataSet_0135.JPEG label 0

name AMD-tinyDataSet_0054.JPEGAMD-tinyDataSet_0005.JPEGAMD-tinyDataSet_0250.JPEGAMD-tinyDataSet_0119.JPEG label 0
name AMD-tinyDataSet_0054.JPEGAMD-tinyDataSet_0005.JPEGAMD-tinyDataSet_0250.JPEGAMD-tinyDataSet_0119.JPEG label 0
name AMD-tinyDataSet_0054.JPEGAMD-tinyDataSet_0005.JPEGAMD-tinyDataSet_0250.JPEGAMD-tinyDataSet_0119.JPEG label 0
name AMD-tinyDataSet_0054.JPEGAMD-tinyDataSet_0005.JPEGAMD-tinyDataSet_0250.JPEGAMD-tinyDataSet_0119.JPEG label 0

name AMD-tinyDataSet_0134.JPEGAMD-tinyDataSet_0038.JPEGAMD-tinyDataSet_0082.JPEGAMD-tinyDataSet_0092.JPEG label 0
name AMD-tinyDataSet_0134.JPEGAMD-tinyDataSet_0038.JPEGAMD-tinyDataSet_0082.JPEGAMD-tinyDataSet_0092.JPEG label 0
name AMD-tinyDataSet_0134.JPEGAMD-tinyDataSet_0038.JPEGAMD-tinyDataSet_0082.JPEGAMD-tinyDataSet_0092.JPEG label 0
name AMD-tinyDataSet_0134.JPEGAMD-tinyDataSet_0038.JPEGAMD-tinyDataSet_0082.JPEGAMD-tinyDataSet_0092.JPEG label 0

name AMD-tinyDataSet_0201.JPEGAMD-tinyDataSet_0012.JPEGAMD-tinyDataSet_0113.JPEGAMD-tinyDataSet_0181.JPEG label 0
name AMD-tinyDataSet_0201.JPEGAMD-tinyDataSet_0012.JPEGAMD-tinyDataSet_0113.JPEGAMD-tinyDataSet_0181.JPEG label 0
name AMD-tinyDataSet_0201.JPEGAMD-tinyDataSet_0012.JPEGAMD-tinyDataSet_0113.JPEGAMD-tinyDataSet_0181.JPEG label 0
name AMD-tinyDataSet_0201.JPEGAMD-tinyDataSet_0012.JPEGAMD-tinyDataSet_0113.JPEGAMD-tinyDataSet_0181.JPEG label 0

name AMD-tinyDataSet_0029.JPEGAMD-tinyDataSet_0240.JPEGAMD-tinyDataSet_0080.JPEGAMD-tinyDataSet_0040.JPEG label 0
name AMD-tinyDataSet_0029.JPEGAMD-tinyDataSet_0240.JPEGAMD-tinyDataSet_0080.JPEGAMD-tinyDataSet_0040.JPEG label 0
name AMD-tinyDataSet_0029.JPEGAMD-tinyDataSet_0240.JPEGAMD-tinyDataSet_0080.JPEGAMD-tinyDataSet_0040.JPEG label 0
name AMD-tinyDataSet_0029.JPEGAMD-tinyDataSet_0240.JPEGAMD-tinyDataSet_0080.JPEGAMD-tinyDataSet_0040.JPEG label 0

...
```

**Comment by rrawther** [comment]
This should work with PR955 and correct use of the app.
./rocAL_basic_test ../../../../data/images/AMD-tinyDataSet/  ../../../../data/images/AMD-tinyDataSet/AMD-tinyDataSet-val.txt 0 0 224 224 1 1
Please make sure this work and close the issue

**Comment by hansely** [comment]
@rrawther 

The tag part is working but I still see the image name as one concatenated string.

```
$ ./rocAL_basic_test ../../../../data/images/AMD-tinyDataSet/ ../../../../data/images/AMD-tinyDataSet-val.txt 0 0 224 224 1 1
>>> Running on CPU
OK: loaded 80 kernels from libvx_rpp.so
Augmented copies count 1
output width 224 output height 896 color planes 3
#### Started test id 0
Available images = 256
>>>>> Going to process 256 images , press a key
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 151
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 18
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 637
name AMD-tinyDataSet_0073.JPEGAMD-tinyDataSet_0224.JPEGAMD-tinyDataSet_0189.JPEGAMD-tinyDataSet_0154.JPEG label 621

name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 367
name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 401
name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 430
name AMD-tinyDataSet_0237.JPEGAMD-tinyDataSet_0096.JPEGAMD-tinyDataSet_0021.JPEGAMD-tinyDataSet_0204.JPEG label 438
```

**Comment by hansely** [comment]
Fixed with PR #955
